File: PycharmProjects/untitled/ex8.py
"""Write a Python program to count the number 4 in a given list."""

def function(num):
    count = 0
    for i in num:
        if i == 4:
            count = count + 1
    return count
print(function([1,2,4,5,4,3]))
print(function([4,4,5,4,6]))

File: c++ code/ex8.cpp
//C++ Program to Generate Multiplication Table
#include<iostream>
#include<string.h>
using namespace std;
int main()
{
    int num;
    cout<<"Enter number : ";
    cin>>num;

    for(int i=1;i<11;i++)
    {
        cout<<num<<" X "<<i<<" = "<<num*i<<"\n";
    }
}